Lib/tkinter/test/support.py

```diff
@@ -115,3 +116,9 @@ def widget_eq(actual, expected):
         if isinstance(expected, (str, tkinter.Widget)):
             return str(actual) == str(expected)
     return False
+
+def ubuntu():
+    try:
+        return "Ubuntu" in os.uname().version
```

Comment:
Nice hack :+1: 

Too bad we don't have ``os-release`` support in the platform module. I had a working patch but I closed it after I was harassed by systemd haters.